.github/workflows/test.yml

```
@@ -12,7 +12,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python: [3.6, 3.7, 3.8, 3.9]
+        python: [3.8, 3.9, "3.10", "3.11"]
```

Nit: For consistency, might be nice to convert them all to strings

```suggestion
        python: ["3.8", "3.9", "3.10", "3.11"]
```